db/compaction/compaction_job.cc

```diff
@@ -1466,10 +1466,24 @@ Status CompactionJob::InstallCompactionResults(
 
 void CompactionJob::RecordCompactionIOStats() {
   RecordTick(stats_, COMPACT_READ_BYTES, IOSTATS(bytes_read));
+  RecordTick(stats_, COMPACT_WRITE_BYTES, IOSTATS(bytes_written));
+  CompactionReason compaction_reason =
+      compact_->compaction->compaction_reason();
+  if (compaction_reason == CompactionReason::kFilesMarkedForCompaction) {
+    RecordTick(stats_, COMPACT_READ_BYTES_DELETE_TRIGGERED,
+               IOSTATS(bytes_read));
+    RecordTick(stats_, COMPACT_WRITE_BYTES_DELETE_TRIGGERED,
```

Comment:
Can we call these `COMPACT_{READ,WRITE}_BYTES_MARKED`? I know we all have been calling it delete-triggered compaction, but actually a deletion-based trigger is just one of the pluggable ways that files can be marked for compaction.